lib/notification.js

```diff
@@ -194,7 +195,7 @@ const NotificationsContainer = styled.div`
   z-index: var(--z-notifications);
   top: 0;
   right: 0;
-  width: 18rem;
```

Comment:
Ah sorry, I was looking at the Notification element itself and didn't notice the `Create Notification`, `Create Success Notification` and `Create Error Notification` buttons that display notifications within a container. If this is the only place where we intend to apply a width to the Notification components then `18rem` does look more correct than `25rem`.

Reply:
👍